$MirOS: ports/devel/autoconf/2.60/patches/patch-lib_m4sugar_m4sugar_m4,v 1.1 2006/07/01 13:30:18 tg Exp $
--- lib/m4sugar/m4sugar.m4.orig	Fri Oct 20 01:34:10 2006
+++ lib/m4sugar/m4sugar.m4	Tue May  8 20:11:14 2007
@@ -1726,12 +1726,17 @@ m4_define([m4_version_compare],
 # --------------------
 m4_include([m4sugar/version.m4])
 
+# m4_PACKAGE_VERSIONNR
+# --------------------
+m4_define([m4_PACKAGE_VERSIONNR],
+	m4_bpatsubst(m4_defn([m4_PACKAGE_VERSION]), [-.*], []))
 
+
 # m4_version_prereq(VERSION, [IF-OK], [IF-NOT = FAIL])
 # ----------------------------------------------------
 # Check this Autoconf version against VERSION.
 m4_define([m4_version_prereq],
-[m4_if(m4_version_compare(m4_defn([m4_PACKAGE_VERSION]), [$1]), -1,
+[m4_if(m4_version_compare(m4_defn([m4_PACKAGE_VERSIONNR]), [$1]), -1,
        [m4_default([$3],
 		   [m4_fatal([Autoconf version $1 or higher is required],
 			     63)])],
